jekyll/_cci2/server/v4.4/installation/phase-1-prerequisites.adoc

```diff
@@ -49,40 +44,71 @@ Download and install the following software before continuing:
 | {helmdiffversion} or greater
 | Helping with `values.yaml` changes and updates
 | Optional, but may help with troubleshooting between releases
+
+| link:https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html[AWS CLI]
```

Comment:
Need some help to add versions and uses here - maybe versions aren't important as they weren't documented previously

Reply:
Latest should be fine

Reply:
👍